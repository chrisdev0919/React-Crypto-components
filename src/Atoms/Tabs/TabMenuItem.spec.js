import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import TabMenuItem from './TabMenuItem'

describe('TabMenuItem component', () => {
  it('renders correctly', () => {
    const component = shallow(<TabMenuItem>MyTabMenuItem</TabMenuItem>)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })

  it('renders correctly when selected', () => {
    const component = shallow(<TabMenuItem selected>MyTabMenuItem</TabMenuItem>)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
