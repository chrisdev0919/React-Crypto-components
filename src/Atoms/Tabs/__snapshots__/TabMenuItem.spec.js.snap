// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabMenuItem component renders correctly 1`] = `
<styled.div>
  MyTabMenuItem
  <styled.div
    visible={false}
  />
</styled.div>
`;

exports[`TabMenuItem component renders correctly when selected 1`] = `
<styled.div>
  MyTabMenuItem
  <styled.div
    visible={true}
  />
</styled.div>
`;
